### ISSUE TITLE
Be able to click on the phone number and call establishment straight from the app

### ISSUE THREAD

**Issue by BigActual** [disAbility-Connections/ion-ap3-child#3]

**Comment by d4mation** [comment]
This would require a RegEx to correctly identify text-only phone numbers and convert them to `tel:` links.

As long as they're consistently in one format throughout the whole site (example: `(517) 867-5309`) then doing so would be easy. But there could be different formats used all over so accounting for them all could be challenging.

Reminder: This could always be done directly within the Post itself when the Establishments are added, and is what I would recommend anyway as it would benefit mobile users who are not using the App. The `tel:` link from the website would then also make it into the App automatically.

**Comment by BigActual** [comment]
At the site level, I assume that we could standardize the data programmatically. What would time and materials look like to make that happen?

**Comment by d4mation** [comment]
Well, yes and no.

The App "runs" a special Theme (This Child Theme, and the Parent Theme provided by AppPresser) which would be what I'd run the RegEx within in my suggestion above. These themes do not effect the in-use Theme on the site if you visit it on a regular web browser.

The best way that it could be done programmatically between both the website and the App would be to loop through all the Posts to swap out the Phone Numbers with Phone Number links and then save the changes. However, that would then rely upon new entries on the site having the Phone Number links being input correctly.

The most reliable way to implement this would be to run the same function, basically, across both the App "Theme" and the Website Theme "on load" so that it doesn't exactly matter how the Phone Number is entered, as it will swap it out with a Phone Number link when the webpage is being shown. 

To handle it either way, I'd need to know what kinds of phone number variations are on the site. Things like:

- 1 (555) 867-5309
- (555) 867-5309
- 555-867-5309

etc.

This way I can reliably write a RegEx that will target _only_ phone numbers and not effect anything else.

**Comment by BigActual** [comment]
(XXX)XXX-XXXX
Go for it.

**Comment by d4mation** [comment]
So every phone number on the site is of the format `(XXX)XXX-XXXX`? Including there being no space between the closing parenthesis and the next number?

**Comment by BigActual** [comment]
No, it needs to be though. I'll ask @disabilityconn to make these adjustments.
